packages/cli/src/supermassive.ts

```diff
@@ -45,6 +46,7 @@ export function supermassive(): Command {
     .option("-cn, --context-name [contextName]", "Context name")
     .option("-ei, --enums-import [enumsImport]", "from where to import enums")
     .option("-l, --legacy", "generate legacy types")
+    .option("-lm, --legacy-models", "do not use models for object types")
```

Comment:
Nit: We don't need 1 letter aliases for options that should only be used in exceptional circumstances.